### PR TITLE
GH-725: added filters and layout settings to test view

### PR DESCRIPTION
**Pull request by jpilgrim**
#725 

As described in issue:

- added filter for showing failures or skipped tests only
- added layout settings: vertical, horizontal or automatic; also hover with test case can be suppressed now
- added flag to omit common prefix of test suites.
- also enabled running of single tests (test case = test method) and rerun failures

### CODE REVIEW

File: plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/resultsview/TestViewFilterHelper.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+			return testStatus == TestStatus.SKIPPED
+					|| testStatus == TestStatus.SKIPPED_FIXME
+					|| testStatus == TestStatus.SKIPPED_IGNORE
+					|| testStatus == TestStatus.SKIPPED_NOT_IMPLEMENTED
+					|| testStatus == TestStatus.SKIPPED_PRECONDITION;
+		}
+		return true;
+	}
+
+}
```
Comment (yoosiba) [inline plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/resultsview/TestViewFilterHelper.java]: add new line

File: plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/resultsview/TestViewLayoutHelper.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+	/**
+	 * Updates actions to reflect proper state.
+	 */
+	public void udpateActions() {
+		for (int i = 0; i < actions.length; ++i)
+			actions[i].setChecked(orientation == actions[i].actionOrientation);
+	}
+
+}
```
Comment (yoosiba) [inline plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/resultsview/TestViewLayoutHelper.java]: add new line

File: plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/resultsview/TestViewFilterHelper.java (hunk 1 above)
Comment (jpilgrim) [reply plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/resultsview/TestViewFilterHelper.java]: done

File: plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/resultsview/TestViewLayoutHelper.java (hunk 2 above)
Comment (jpilgrim) [reply plugins/org.eclipse.n4js.tester.ui/src/org/eclipse/n4js/tester/ui/resultsview/TestViewLayoutHelper.java]: done

**Comment by jpilgrim** [comment]
Detected some problems:
- config is not saved when openend via shortcut (and modified and OK was pressed)
- relaunch etc. is not working reliably

**Comment by jpilgrim** [comment]
@yoosiba Could you please have another look -- I also added support for running single tests and failures only.

**Pull request merged**